stable/openldap/README.md

```diff
@@ -23,27 +23,34 @@ We use the docker images provided by https://github.com/osixia/docker-openldap.
-| `image.pullPolicy`              | Container pull policy                                         | `IfNotPresent`     |
-| `containerName`                 | Name of the container in the Pod                              | `slapd`            |
-| `service.type`                  | Service type                                                  | `ClusterIP`        |
-| `service.ldapPort`              | External service port for LDAP                                | `389`              |
-| `service.sslLdapPort`           | External service port for SSL+LDAP                            | `636`              |
-| `env`                           | List of key value pairs as env variables to be sent to the docker image. See https://github.com/osixia/docker-openldap for available ones | `[see values.yaml]`  |
-| `adminPassword`                 | Password for admin user. Unset to auto-generate the password  | None               |
-| `configPassword`                | Password for config user. Unset to auto-generate the password | None               |
-| `customLdifFiles`               | Custom ldif files to seed the LDAP server. List of filename -> data pairs | None   |
-| `persistence.enabled`           | Whether to use PersistentVolumes or not                       | `false`            |
-| `persistence.storageClass`      | Storage class for PersistentVolumes.                          | `<unset>`          |
-| `persistence.accessMode`        | Access mode for PersistentVolumes                             | `ReadWriteOnce`    |
-| `persistence.size`              | PersistentVolumeClaim storage size                            | `8Gi`              |
-| `resources`                     | Container resource requests and limits in yaml                | `{}`               |
-| `test.image.repository`         | Test container image requires bats framework                  | `dduportal/bats`   |
-| `test.image.tag`                | Test container tag                                            | `0.4.0`            |
+| Parameter                          | Description                                                               | Default           |
+| ---------------------------------- | ------------------------------------------------------------------------- | ------------------|
+| `replicaCount`                     | Number of replicas                                                        | `1`               |
+| `image.repository`                 | Container image repository                                                | `osixia/openldap` |
+| `image.tag`                        | Container image tag                                                       | `1.1.10`          |
+| `image.pullPolicy`                 | Container pull policy                                                     | `IfNotPresent`    |
+| `extraLabels`                      | Labels to add to the Resources                                            | `{}`              |
```

Comment:
This is used by only a handful of charts that I can find. Seems useful, but not sure whether this is the convention.